axelor-account/src/main/resources/views/Account.xml

```diff
@@ -120,8 +120,9 @@
           <field name="vatSystemSelect" colSpan="4"
             showIf="isTaxAuthorizedOnMoveLine &amp;&amp; (accountType.technicalTypeSelect == 'immobilisation' || accountType.technicalTypeSelect == 'charge' || accountType.technicalTypeSelect == 'income')"/>
         </panel>
-        <panel name="analyticSettingsPanel" title="Analytic" colSpan="12"
-          hideIf="!accountType || accountType.technicalTypeSelect == 'view'">
+        <panel
+          if="__config__.app.getApp('account').getManageAnalyticAccounting() || (!accountType &amp;&amp; accountType.technicalTypeSelect != 'view')"
```

Comment:
I think it's : 
`if="__config__.app.getApp('account').getManageAnalyticAccounting() &amp;&amp; accountType.technicalTypeSelect != 'view'"`

Reply:
Solved